test/v2/controllers/CreateAmendForeignPropertyAnnualSubmissionControllerSpec.scala

```diff
@@ -48,71 +48,12 @@ class CreateAmendForeignPropertyAnnualSubmissionControllerSpec
     with MockIdGenerator
     with CreateAmendForeignPropertyAnnualSubmissionFixture {
 
-  private val nino          = "AA123456A"
-  private val businessId    = "XAIS12345678910"
-  private val taxYear       = "2019-20"
-  private val correlationId = "X-123"
-
-  trait Test {
-    val hc: HeaderCarrier = HeaderCarrier()
-
-    val controller = new CreateAmendForeignPropertyAnnualSubmissionController(
-      authService = mockEnrolmentsAuthService,
-      lookupService = mockMtdIdLookupService,
-      parser = mockCreateAmendForeignPropertyAnnualSubmissionRequestParser,
-      service = mockService,
-      auditService = mockAuditService,
-      hateoasFactory = mockHateoasFactory,
-      cc = cc,
-      idGenerator = mockIdGenerator
-    )
-
-    MockMtdIdLookupService.lookup(nino).returns(Future.successful(Right("test-mtd-id")))
-    MockedEnrolmentsAuthService.authoriseUser()
-    MockIdGenerator.getCorrelationId.returns(correlationId)
-  }
-
-  private val testHateoasLink = Link(href = s"/individuals/business/property/$nino/$businessId/annual/$taxYear", method = GET, rel = "self")
-
-  val hateoasResponse: JsValue = Json.parse(
-    s"""
-       |{
-       |   "links": [
-       |      {
-       |         "href": "/individuals/business/property/$nino/$businessId/annual/$taxYear",
-       |         "method": "GET",
-       |         "rel": "self"
-       |      }
-       |   ]
-       |}
-    """.stripMargin
-  )
-
-  def event(auditResponse: AuditResponse): AuditEvent[GenericAuditDetail] =
-    AuditEvent(
-      auditType = "CreateAmendForeignPropertyAnnualSubmission",
-      transactionName = "create-amend-foreign-property-annual-submission",
-      detail = GenericAuditDetail(
-        versionNumber = "2.0",
-        userType = "Individual",
-        agentReferenceNumber = None,
-        params = Json.obj("nino" -> nino, "businessId" -> businessId, "taxYear" -> "2019-20", "request" -> requestJson),
-        correlationId = correlationId,
-        response = auditResponse
-      )
-    )
-
-  private val requestJson = createAmendForeignPropertyAnnualSubmissionRequestBodyMtdJson
-
-  val body: CreateAmendForeignPropertyAnnualSubmissionRequestBody = createAmendForeignPropertyAnnualSubmissionRequestBody
-
-  private val rawData = CreateAmendForeignPropertyAnnualSubmissionRawData(nino, businessId, taxYear, requestJson)
-  private val request = CreateAmendForeignPropertyAnnualSubmissionRequest(Nino(nino), businessId, TaxYear.fromMtd(taxYear), body)
+  private val businessId = "XAIS12345678910"
+  private val taxYear    = "2019-20"
 
   "handleRequest" should {
```

Comment:
Would be better to update the wording of this test to:

`
"CreateAmendForeignPropertyAnnualSubmissionController" should {
    "return a successful response with status 200 (OK)" when {
      "the request received is valid" in new Test {

}}}
`